cico_run_tests.sh

```diff
@@ -5,5 +5,6 @@
 cico_setup;
 
 run_tests_without_coverage;
+run_e2e_tests;
```

Comment:
It would be nice if you can make this a separate job instead (e.g. in `cico_run_e2e_tests.sh`). Then our unit+integration tests come back more quickly while the integration tests can continue running. Also we avoid polluting the unit tests with messages from E2E tests.